src/UI/CardPage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import styles from "./CardPage.module.css";
+import { CardHomeInterface } from "../interfaces/CardHomeInterface";
+import { Link } from "react-router-dom";
+
+const CardPage = (props: CardHomeInterface) => {
+  return (
+    <div className={styles.card}>
+      <Card
+        className="bg-dark text-white"
+        style={{ width: "12vw", height: "23em" }}
+      >
+        <Card.Img variant="top" style={{ height: "50%" }} src={props.images} />
```

Comment:
Same here... is 50% going to work across all devices?

Reply:
also addressed in the next pull request